.github/workflows/ci-e2e-tests.yml

```diff
@@ -90,34 +90,74 @@ jobs:
         run: |
           KARDINAL_CLI_DEV_MODE=TRUE /tmp/kardinal-cli manager deploy local-minikube
 
+          # Check that the three kardinal manager service pods are running and ready
+          while [ $(kubectl get pods -n default --no-headers -o custom-columns=NAMESPACE:metadata.namespace,POD:metadata.name,PodIP:status.podIP,READY-true:status.containerStatuses[*].ready | grep "true" | wc -l) -ne 3 ]
+          do
+              echo "Waiting for kardinal manager pods to run..."
+              kubectl get pods -n default -o custom-columns=NAMESPACE:metadata.namespace,POD:metadata.name,PodIP:status.podIP,READY-true:status.containerStatuses[*].ready
+              ((c++)) && ((c==12)) && exit 1
+              sleep 10
+          done
+          apps=$(kubectl get pods -n default -o custom-columns=:metadata.labels.app | tr " " "\n" | sort -g | tr "\n" " " | xargs)
+          echo ${apps}
+          if [ "${apps}" != "kardinal-manager trace-router trace-router-redis" ]; then exit 1; fi
+          
+          # Check for errors in the kardinal manager logs
+          if kubectl logs -n default -l dev.kardinal.app-id=kardinal-manager | grep "ERRO"
```

Comment:
Wouldn't grep return 1 if it doesn't find the "ERRO" string?
And you can add the `-q` flag to set the quit mode to not print the output

Reply:
The if condition is fulfilled when grep exits with 0 (match found).  I will add the quiet flag.